integration_tests/dbt_project/macros/materializations.sql

```diff
@@ -0,0 +1,85 @@
+{# Table#}
+
+{% materialization table, default %}
+  {% do return(elementary.materialization_table_default()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="snowflake", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_table_snowflake()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="bigquery", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_table_bigquery()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="spark", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_table_spark()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="databricks", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_table_databricks()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="redshift", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_table_redshift()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="athena", supported_languages=["sql"] %}
+  {% do return(elementary.materialization_table_athena()) %}
+{% endmaterialization %}
+
+{% materialization table, adapter="trino", supported_languages=["sql"] %}
+  {% do return(elementary.materialization_table_trino()) %}
+{% endmaterialization %}
+
+{# Incremental #}
+
+{% materialization incremental, default %}
+  {% do return(elementary.materialization_incremental_default()) %}
+{% endmaterialization %}
+
+{% materialization incremental, adapter="snowflake", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_incremental_snowflake()) %}
+{% endmaterialization %}
+
+{% materialization incremental, adapter="bigquery", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_incremental_bigquery()) %}
+{% endmaterialization %}
+
+{% materialization incremental, adapter="spark", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_incremental_spark()) %}
+{% endmaterialization %}
+
+{% materialization incremental, adapter="databricks", supported_languages=["sql", "python"] %}
+  {% do return(elementary.materialization_incremental_databricks()) %}
+{% endmaterialization %}
+
+{% materialization incremental, adapter="athena", supported_languages=["sql"] %}
+  {% do return(elementary.materialization_incremental_athena()) %}
+{% endmaterialization %}
+
+{% materialization incremental, adapter="trino", supported_languages=["sql"] %}
+  {% do return(elementary.materialization_incremental_trino()) %}
+{% endmaterialization %}
+
+{# Test #}
+
+{% materialization test, default %}
+  {% if var('enable_elementary_test_materialization', false) %}
```

Comment:
Wanted the integration tests to run by default without the test materialization, so we can catch exactly the cases that require it to work properly.
Thought to do the same for the other materializations but it was a bit of a hassle so I didn't.